### ISSUE TITLE
Is there a pretrained mobilenet weights in model zoo?

### ISSUE THREAD

**Issue by BoXiao123** [apache/incubator-mxnet#8859]
I am looking for a pretrained mobilenet weights. But it seems not exist in the model zoo. Will mxnet provide it in the model zoo?

**Comment by edmBernard** [comment]
For the regular Mxnet API there is a mobilenet pretrained model convert from caffe model [here](https://github.com/KeyKy/mobilenet-mxnet)

**Comment by shuokay** [comment]
gluon have a pretrained mobilenet, and also can convert to symbol pretrained model.

**Comment by edmBernard** [comment]
Are you sure there is pretrained mobilenet for gluon ? 

I was not able to get it: 
```python
>>> import mxnet as mx
>>> from mxnet import gluon
>>> gluon.model_zoo.vision.mobilenet1_0(pretrained=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.5/dist-packages/mxnet-0.12.1-py3.5.egg/mxnet/gluon/model_zoo/vision/mobilenet.py", line 115, in mobilenet1_0
    return get_mobilenet(1.0, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/mxnet-0.12.1-py3.5.egg/mxnet/gluon/model_zoo/vision/mobilenet.py", line 100, in get_mobilenet
    net.load_params(get_model_file('mobilenet%s'%version_suffix, root=root), ctx=ctx)
  File "/usr/local/lib/python3.5/dist-packages/mxnet-0.12.1-py3.5.egg/mxnet/gluon/model_zoo/model_store.py", line 80, in get_model_file
    short_hash=short_hash(name))
  File "/usr/local/lib/python3.5/dist-packages/mxnet-0.12.1-py3.5.egg/mxnet/gluon/model_zoo/model_store.py", line 58, in short_hash
    raise ValueError('Pretrained model for {name} is not available.'.format(name=name))
ValueError: Pretrained model for mobilenet1.0 is not available.
```

**Comment by shuokay** [comment]
Sorry, the is no mobilenet params file in model zoo.
You can use the model convert from caffe. Just load the params file with mx.ndarray.load, and change key values to the name of symbols.

**Comment by BoXiao123** [comment]
thx, @shuokay 
I'll try to use that one convert from caffe.

**Comment by szha** [comment]
This will be resolved in #9215

**Comment by szha** [comment]
mobilenet pre-trained model is now provided thanks to @hetong007
